src/app/components/checkin/checkin.controller.js

```diff
@@ -24,38 +24,50 @@
     $scope.isCurrentUser = true;
     $scope.lists = [];
 
-    var queryCallback = function () {
```

Comment:
@guillaumev I've removed this as couldn't see why would want to get the lists before the user searches for one, let me know if there was a reason I'm missing